spacy/training/loggers.py

```diff
@@ -23,13 +26,35 @@ def setup_table(
     return final_cols, final_widths, ["r" for _ in final_widths]
 
 
-@registry.loggers("spacy.ConsoleLogger.v1")
-def console_logger(progress_bar: bool = False):
+@registry.loggers("spacy.ConsoleLogger.v2")
+def console_logger(
+    progress_bar: bool = False,
+    print_progress: bool = True,
+    output_file: Union[str, Path] = None,
+):
+    """The ConsoleLogger.v2 prints out training logs in the console and saves them into a jsonl file, with both options being optional.
```

Comment:
```suggestion
    """The ConsoleLogger.v2 prints out training logs in the console and/or saves them to a jsonl file.
```

Reply:
"in" the console? Shouldn't it be "on" the console?